3scale-image-streams.yml

```diff
@@ -0,0 +1,37 @@
+kind: ImageStreamList
+apiVersion: v1
+metadata: {}
+items:
+- kind: ImageStream
+  apiVersion: v1
+  metadata:
+    name: "apicast-gateway"
+    annotations:
+      openshift.io/display-name: "3scale APIcast API Gateway"
+  spec:
+    tags:
+    - name: latest
+      annotations:
+        openshift.io/display-name: "3scale APIcast API Gateway (latest)" 
+        openshift.io/provider-display-name: "Red Hat, Inc."
+        iconClass: "icon-3scale"
+        description: >- 
+          "3scale's APIcast API Gateway is an OpenResty application which consists of two parts: 
```

Comment:
http://yaml-multiline.info shows that this could be without the quotes. And you can control if you want newlines on the end or convert them to spaces.